app/routes/individual_response.py

```diff
@@ -0,0 +1,156 @@
+from flask import Blueprint, g, redirect, request, url_for
+from flask_login import current_user, login_required
+from structlog import get_logger
+
+from app.authentication.no_token_exception import NoTokenException
+from app.globals import get_metadata, get_session_store
+from app.helpers.language_helper import handle_language
+from app.helpers.schema_helpers import with_schema
+from app.helpers.session_helpers import with_questionnaire_store
+from app.helpers.template_helper import render_template
+from app.utilities.schema import load_schema_from_session_data
+from app.views.handlers.individual_response import (
+    IndividualResponseHandler,
+    IndividualResponseHowHandler,
+    IndividualResponsePostAddressConfirmHandler,
+)
+from app.questionnaire.location import InvalidLocationException
+
+logger = get_logger()
+
+individual_response_blueprint = Blueprint(
+    name="individual_response", import_name=__name__, url_prefix="/individual-response/"
+)
+
+
+@individual_response_blueprint.before_request
+def before_individual_response_request():
```

Comment:
The IR routes are missing the language toggle.

Reply:
Because the test schema only has an 'en' version